experimental/unitsconversion/src/helpers.rs

```diff
@@ -2,10 +2,57 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+use std::str::FromStr;
+
+use num_bigint::BigUint;
+use num_traits::{Zero, One};
+
+use crate::Error;
+
+
+pub fn gcd_big_unit(a: BigUint, b: BigUint) -> Result<BigUint, Error> {
```

Comment:
note that `BigUint` comes with a (more efficient) implementation already: https://docs.rs/num-bigint/latest/num_bigint/struct.BigUint.html#method.gcd